profiles/sink/Sink_AudioSettings.yaml

```diff
@@ -61,7 +61,7 @@ dsAudio:
             # dsAUDIO_CMP_MAX    = 0x04   ///< Out of range
             compressions: [0x00, 0x01, 0x02, 0x03]  # NONE, LIGHT, MEDIUM, HEAVY
             # Number of supported Stereo modes
-            number_of_supported_stereo_modes: 1
+            number_of_supported_stereo_modes: 0
```

Comment:
unclear why this is being modified, this is the default settings, which are enable everything.

The specific platform test settings is not this file.